Cleaning DB/1 Table cleaning.sql

```diff
@@ -0,0 +1,35 @@
+use CleaningDB 
+go 
+
+drop table if exists dbo.Customer 
+go
+create table dbo.Customer(
+    CustomerID int not null identity primary key,
+    FirstName varchar(20) not null 
+            constraint c_Customer_first_name_cannot_be_blank check(FirstName <> ''),
+    LastName varchar(20) not null 
+            constraint c_Customer_last_name_cannot_be_blank check(LastName <> ''),
+    PhoneNum char(12) not null 
+            constraint c_Customer_phone_num_cannot_be_blank check(PhoneNum <> ''),
+    HomeAddress varchar(40) not null 
+            constraint c_Customer_home_address_cannot_be_blank check(HomeAddress <> '')
+            constraint u_Customer_home_address_must_be_unique unique,
+    Frequency varchar(10)
+            constraint c_Customer_frequency_cannot_be_blank check(Frequency <> ''),
+    RatePerHour decimal(5,2),
+    AvgTime int not null 
```

Comment:
When dealing with an int, we need to validate that it is not 0, not that it is not blank